src/FluentHub.Uwp/UserControls/SearchResultSidebar.xaml

```diff
@@ -0,0 +1,128 @@
+
+                <muxc:NavigationViewItem Content="Issues" Tag="issues">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Issues.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem
+                    Content="Discussions"
+                    IsEnabled="False"
+                    Tag="discussions">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Discussions.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem
+                    Content="Packages"
+                    IsEnabled="False"
+                    Tag="packages">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Packages.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem
+                    Content="Marketplace"
+                    IsEnabled="False"
+                    Tag="marketplace">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Marketplace.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem
+                    Content="Topics"
+                    IsEnabled="False"
+                    Tag="topics">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Topic.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem
+                    Content="Wikis"
+                    IsEnabled="False"
+                    Tag="wikis">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Wiki.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+
+                <muxc:NavigationViewItem Content="Users" Tag="users">
+                    <muxc:NavigationViewItem.Icon>
+                        <muxc:ImageIcon Source="/Assets/Icons/Profile.png" />
+                    </muxc:NavigationViewItem.Icon>
+                </muxc:NavigationViewItem>
+            </muxc:NavigationView.MenuItems>
+
+        </muxc:NavigationView>
+
+        <TextBlock
+            Grid.Row="2"
+            Margin="32,20"
+            Foreground="{ThemeResource SystemFillColorCautionBrush}"
+            IsTextSelectionEnabled="True"
+            Style="{StaticResource BodyTextBlockStyle}"
+            Text="Currently, repos, code, issues, users searching is only available." />
```

Comment:
```diff
+        <TextBlock
+           Grid.Row="2"
+            Margin="32,20"
+            Foreground="{ThemeResource SystemFillColorCautionBrush}"
+            IsTextSelectionEnabled="True"
+            Style="{StaticResource BodyTextBlockStyle}"
+            Text="Currently, repos, code, issues, users searching is only available." />            
+            Grid.Row="2"
+            Margin="32,20"
+            Foreground="{ThemeResource SystemFillColorCautionBrush}"
+            IsTextSelectionEnabled="True"
+            Style="{StaticResource BodyTextBlockStyle}"
+            Text="Currently, repos, code, issues, users searching is only available." />
```
Should be an `InfoBar`

Reply:
Maybe should be deleted. It's very difficult. When communicating these API limitations to the user, I'm thinking about whether to write it for each page, write it on the settings page, or just write it on the official website as a document.